### PR TITLE
[DOC] Fix links for mounting OSS content to GET

### PR DESCRIPTION
**Pull request by knylander-grafana**
**What this PR does**:

Corrects links and references so the intro, TraceQL, and metrics-generator content can be mounted into GET documentation. 

**Which issue(s) this PR fixes**:
Part of https://github.com/grafana/enterprise-traces/issues/1365
Blocks https://github.com/grafana/website/pull/22927

**Checklist**
- [ ] Tests updated
- [X] Documentation added
- [ ] `CHANGELOG.md` updated - the order of entries should be `[CHANGE]`, `[FEATURE]`, `[ENHANCEMENT]`, `[BUGFIX]`

### CODE REVIEW

File: docs/sources/tempo/metrics-generator/service-graph-view.md [hunk 1]
```diff
@@ -5,6 +5,17 @@ description: Grafana's service graph view utilizes metrics generated by the metr
...
+  enable-service-graphs:
+    - pattern: /docs/tempo/
+      destination: https://grafana.com/docs/tempo/<TEMPO_VERSION>/metrics-generator/service_graphs/enable-service-graphs/
+    - pattern: /docs/enterprise-traces/
+      destination: https://grafana.com/docs/enterprise-traces/<ENTERPRISE_TRACES_VERSION>/metrics-generator/service_graphs/enable-service-graphs/
+  span-metrics:
+    - pattern: /docs/tempo/
+      destination: https://grafana.com/docs/tempo/<TEMPO_VERSION>/metrics-generator/span-metrics/
+    - pattern: /docs/enterprise-traces/
+      destination: https://grafana.com/docs/enterprise-traces/<ENTERPRISE_TRACES_VERSION>/metrics-generator/span-metrics/
```
Comment (jdbaldry) [inline docs/sources/tempo/metrics-generator/service-graph-view.md:18]: The page appears to use underscores:

```suggestion
  span-metrics:
    - pattern: /docs/tempo/
      destination: https://grafana.com/docs/tempo/<TEMPO_VERSION>/metrics-generator/span_metrics/
    - pattern: /docs/enterprise-traces/
      destination: https://grafana.com/docs/enterprise-traces/<ENTERPRISE_TRACES_VERSION>/metrics-generator/span_metrics/
```

File: docs/sources/tempo/traceql/metrics-queries/functions.md [hunk 2]
```diff
@@ -22,16 +22,16 @@ These functions can be added as an operator at the end of any TraceQL query.
...
+: Returns the minimum value for the specified attribute across all matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs#traceql-metrics)).
 
 `max_over_time`
-: Returns the maximum value for the specified attribute across all matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics)).
+: Returns the maximum value for the specified attribute across all matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs#traceql-metrics)).
 
 `avg_over_time`
-: Returns the average value for the specified attribute across all matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics)).
+: Returns the average value for the specified attribute across all matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs#traceql-metrics)).
```
Comment (jdbaldry) [inline docs/sources/tempo/traceql/metrics-queries/functions.md:34]: These changes can be discarded, they make no difference to the final URL.

File: docs/sources/tempo/traceql/metrics-queries/functions.md [hunk 3]
```diff
@@ -85,8 +85,7 @@ spans than `/api/happy`, for example.
 
 The `count_over_time()` function counts the number of matching spans per time interval.
 The time interval that the count will be computed over is set by the `step` parameter.
-For more information, refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics).
-
+For more information, refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs#traceql-metrics).
```
Comment (jdbaldry) [inline docs/sources/tempo/traceql/metrics-queries/functions.md:88]: These changes can be discarded, they make no difference to the final URL.

File: docs/sources/tempo/traceql/metrics-queries/functions.md [hunk 4]
```diff
@@ -109,7 +108,7 @@ The time interval that the maximum is computed over is set by the `step` paramet
 The `avg_over_time()` function lets you aggregate numerical values by computing the maximum value of them, such as the all important span duration.
 The time interval that the maximum is computer over is set by the `step` parameter.
 
-For more information, refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics).
+For more information, refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs#traceql-metrics).
```
Comment (jdbaldry) [inline docs/sources/tempo/traceql/metrics-queries/functions.md:111]: These changes can be discarded, they make no difference to the final URL.

File: docs/sources/tempo/traceql/metrics-queries/functions.md [hunk 5]
```diff
@@ -186,7 +185,7 @@ You could use a similar query to know what the 50th percentile and 95th percenti
 The `compare` function is used to split a set of spans into two groups: a selection and a baseline.
 It returns time-series for all attributes found on the spans to highlight the differences between the two groups.
 
-This is a powerful function that's best understood by using the [**Comparison** tab in Explore Traces](https://grafana.com/docs/grafana/<GRAFNA_VERSION>/explore/simplified-exploration/traces/investigate/#comparison).
+This is a powerful function that's best understood by using the [**Comparison** tab in Explore Traces](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/simplified-exploration/traces/investigate#comparison).
```
Comment (jdbaldry) [inline docs/sources/tempo/traceql/metrics-queries/functions.md:188]:
```suggestion
This is a powerful function that's best understood by using the [**Comparison** tab in Explore Traces](https://grafana.com/docs/grafana/<GRAFANA_VERSION>/explore/simplified-exploration/traces/investigate/#comparison).
```

File: docs/sources/tempo/metrics-generator/service-graph-view.md (hunk 1 above)
Comment (knylander-grafana) [reply docs/sources/tempo/metrics-generator/service-graph-view.md:18]: Good catch.

File: docs/sources/tempo/traceql/metrics-queries/functions.md [hunk 6]
```diff
@@ -22,16 +22,16 @@ These functions can be added as an operator at the end of any TraceQL query.
...
+: Counts the number of matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs)).
 
 `min_over_time`
-: Returns the minimum value for the specified attribute across all matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics)).
+: Returns the minimum value for the specified attribute across all matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs#traceql-metrics)).
 
 `max_over_time`
-: Returns the maximum value for the specified attribute across all matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics)).
+: Returns the maximum value for the specified attribute across all matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs#traceql-metrics)).
```
Comment (knylander-grafana) [inline docs/sources/tempo/traceql/metrics-queries/functions.md:31]:
```suggestion
: Returns the maximum value for the specified attribute across all matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics)).
```

File: docs/sources/tempo/traceql/metrics-queries/functions.md [hunk 7]
```diff
@@ -22,16 +22,16 @@ These functions can be added as an operator at the end of any TraceQL query.
 : Calculates the number of matching spans per second
 
 `count_over_time`
-: Counts the number of matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics)).
+: Counts the number of matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs)).
 
 `min_over_time`
-: Returns the minimum value for the specified attribute across all matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics)).
+: Returns the minimum value for the specified attribute across all matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs#traceql-metrics)).
```
Comment (knylander-grafana) [inline docs/sources/tempo/traceql/metrics-queries/functions.md:28]:
```suggestion
: Returns the minimum value for the specified attribute across all matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics)).
```

File: docs/sources/tempo/traceql/metrics-queries/functions.md (hunk 3 above)
Comment (knylander-grafana) [reply docs/sources/tempo/traceql/metrics-queries/functions.md:88]:
```suggestion
For more information, refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics).
```

File: docs/sources/tempo/traceql/metrics-queries/functions.md (hunk 4 above)
Comment (knylander-grafana) [inline docs/sources/tempo/traceql/metrics-queries/functions.md:111]:
```suggestion
For more information, refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics).
```

File: docs/sources/tempo/traceql/metrics-queries/functions.md [hunk 8]
```diff
@@ -31,7 +31,7 @@ These functions can be added as an operator at the end of any TraceQL query.
 : Returns the maximum value for the specified attribute across all matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics)).
 
 `avg_over_time`
-: Returns the average value for the specified attribute across all matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics)).
+: Returns the average value for the specified attribute across all matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs#traceql-metrics)).
```
Comment (knylander-grafana) [inline docs/sources/tempo/traceql/metrics-queries/functions.md:34]:
```suggestion
: Returns the average value for the specified attribute across all matching spans per time interval (refer to the [`step` API parameter](https://grafana.com/docs/tempo/<TEMPO_VERSION>/api_docs/#traceql-metrics)).
```

File: docs/sources/tempo/traceql/metrics-queries/functions.md (hunk 2 above)
Comment (knylander-grafana) [reply docs/sources/tempo/traceql/metrics-queries/functions.md]: reverted the changes.

File: docs/sources/tempo/traceql/metrics-queries/functions.md (hunk 4 above)
Comment (knylander-grafana) [reply docs/sources/tempo/traceql/metrics-queries/functions.md]: Done.

**Review by jdbaldry — APPROVED** [review]
Nothing jumps out after a second scan. Thanks for incorporating my suggestions :)

**Review by joe-elliott — APPROVED** [review]

**Pull request merged**